apparmor.d/groups/kde/kscreenlocker-greet

```diff
@@ -57,6 +68,7 @@ profile kscreenlocker-greet @{exec_path} {
 
   owner @{HOME}/.face.icon r,
   owner @{HOME}/.xsession-errors w,
+  owner @{HOME}/@{XDG_PICTURES_DIR}/{,**} r,
```

Comment:
Use `owner @{user_pictures_dirs}/{,**} r,` so that it can also support mounted disks